src/main/java/org/elasticsearch/SecureSM.java

```diff
@@ -170,28 +162,13 @@ protected void checkThreadGroupAccess(ThreadGroup g) {
   // exit permission logic
   @Override
   public void checkExit(int status) {
-    if (allowTestExit) {
-      checkTestExit(status);
-    } else {
-      throw new SecurityException("exit(" + status + ") not allowed by system policy");
-    }
+    checkExit(status, packagesThatCanExit);
   }
-
-  static final String TEST_RUNNER_PACKAGES[] = {
-    // surefire test runner
-    "org.apache.maven.surefire.booter.",
-    // junit4 test runner
-    "com.carrotsearch.ant.tasks.junit4.",
-    // eclipse test runner
-    "org.eclipse.jdt.internal.junit.runner.",
-    // intellij test runner
-    "com.intellij.rt.execution.junit."
-  };
```

Comment:
How about a static method instead, so it can return an array. I don't like a field because it means pulling in java.util.List and other things. Frankly. we shouldn't even be pulling in java.util.Objects as-is. We have to be (more) minimal to protect ourselves against bugs like https://bugs.openjdk.java.net/browse/JDK-8155090 and https://bugs.openjdk.java.net/browse/JDK-8155659